integration/kotlinx-coroutines-jdk8/src/main/kotlin/kotlinx/coroutines/Time.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package kotlinx.coroutines
```

Comment:
Let's put it in `kotlinx.coroutines.experimental.time` package (we should not have a package without `experimental` and we cannot use the same package name as in core module due to Java9 module system restrictions)